integration/quarkus/runtime/pom.xml

```diff
@@ -78,6 +78,25 @@
             </resource>
         </resources>
         <plugins>
+            <plugin>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
```

Comment:
@aloubyansky can you check I did things properly? I checked the metadata and they look fine now.